lib/membrane/rtp/parser.ex

```diff
@@ -52,7 +52,7 @@ defmodule Membrane.RTP.Parser do
 
   @impl true
   def handle_init(opts) do
-    {:ok, %{rtcp_output_pad: nil, secure?: opts.secure?}}
+    {:ok, %{rtcp_output_pad: nil, secure?: opts.secure?, has_session_output?: false}}
```

Comment:
I suppose that `has_session_output` is set whenever `rtcp_output_pad` is not nil

Reply:
I don't *think* this is true. In all cases I've seen so far, `rtcp?` is true, whether or not an output has been added to to the session bin.

https://github.com/membraneframework/membrane_rtp_plugin/blob/c24abf0e5480564021417e90f0e55b38aaf5acf6/lib/membrane/rtp/session_bin.ex#L267-L269
https://github.com/membraneframework/membrane_rtp_plugin/blob/c24abf0e5480564021417e90f0e55b38aaf5acf6/lib/membrane/rtp/session_bin.ex#L279-L301

Reply:
I think that even if the `session_bin` does not have an output pad, the `rtp -> rtcp -> ssrc_router` links are established.

Reply:
> In all cases I've seen so far, rtcp? is true, whether or not an output has been added to to the session bin.

Hmm, that's weird, because `rtcp?` seems to be set only when the rtcp output of the session bin is connected. IMHO that wouldn't work otherwise, because within `if rtcp?` we're linking to the rtcp output of the session bin.